index.html

```diff
@@ -0,0 +1,36 @@
+<!DOCTYPE html>
+<html lang="en" ng-app="ToDo">
+<head>
+  <meta charset="utf-8">
+  <title>To-Do</title>
+  <style>
+  .done{text-decoration: line-through; color: #ccc;}
+  </style>
+  <link rel="stylesheet" href="bower_components/bootstrap/dist/css/bootstrap.css">
+  <link rel="stylesheet" type="text/css" href="style/style.css">
+  <script src="bower_components/jquery/dist/jquery.js"></script>
+  <script src="bower_components/angular/angular.js"></script>
+  <script src="bower_components/angular-resource/angular-resource.js"></script>
+  <script src="js/app.js"></script>
+  <script src="js/toDoController.js"></script>
+</head>
+
+<body>
+  <h1>To-Do</h1>
+  <div ng-controller="ToDoController">
+    <form name="frm" ng-submit="addToDo()">
+      <input type="text" id="newToDo" ng-model="newToDo" placeholder="What needs to be done?"required>
+      <button ng-disabled="frm.$invalid">Add</button>
+    </form>
+    <br>
+      <button ng-click="clearCompleted()">Clear Completed</button>
+    <br>
+      <ul id="list">
+          <li ng-repeat="todo in todos">
+            <input type="checkbox" ng-model="todo.done">
```

Comment:
You can enhance the checkbox functionality by implementing Angular's checkbox directives (see link: https://docs.angularjs.org/api/ng/input/input%5Bcheckbox%5D)